license/license.md

```diff
@@ -67,3 +66,19 @@ Here is a handy table that explains the events leading up to and after license e
 ✅ - full support
 
 <img src="/_static/favicon.ico" alt="gravwell managed" width="20"/> - Gravwell managed
+
+## Free Tier Feature Availability Warnings
```

Comment:
```suggestion
## Free Edition Feature Availability Warnings
```

Kris called a Free edition up above (in the License Expiration section), so I am suggesting we use the same language here, but I don't think it _really_ matters. This is just a suggestion.